test/cypress/integration/group-paying.spec.js

```diff
@@ -118,7 +122,7 @@ describe('Group Payments', () => {
       cy.getByDT('amount').should('contain', '$71.43')
       cy.getByDT('subtitle').should('contain', `Sent to user2-${userId}`)
 
-      cy.getByDT('details').find('li:nth-child(2)').should('contain', humanDate(timeStart, { month: 'long', year: 'numeric' }))
+      cy.getByDT('details').find('li:nth-child(2)').should('contain', humanDate(dateToMonthstamp(new Date(timeStart)), { month: 'long', year: 'numeric' }))
```

Comment:
Apparently the test failed because it didn't take account the transformation happening in `L:55 of PaymentDetail.vue` Where the date becomes a monthstamp.
This change made all the tests pass in my local.